sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.http.rest.PagedFlux;
+import reactor.core.publisher.Mono;
+
+import java.util.Map;
+
+@ServiceClient(
+    builder = TableClientBuilder.class,
+    isAsync = true)
+public class TableAsyncClient {
+    String tableName;
+
+
+    TableAsyncClient(String tableName) {
+        this.tableName = tableName;
+    }
+
```

Comment:
nit: unnecessary empty lines. There are several places where you have 2 empty lines instead of one.